code/script.js

```diff
@@ -254,90 +255,119 @@ const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+  generateBoard()
+  setSecret()
+  winOrLose.style.display = ''
```

Comment:
If you decide to make changes I suggested at the end of the code, you need to make some here as well:
```
winOrLose.style.display = 'none'
board.style.display='flex'
```